### PR TITLE
Add global styles for shadow and bg

### PR DESCRIPTION
**Pull request by adamish717**
Adds textured background and shadows to container boxes

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://adamish717-global-add.sinetheta-steam-hours-31289418a5192091c5e3.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://adamish717-global-add.sinetheta-steam-hours-31289418a5192091c5e3.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/theme/_global.scss [hunk 1]
```diff
@@ -5,4 +5,12 @@
   border-radius: $border-radius-base;
   border: 1px solid $gray-lighter;
   margin-bottom: 2em;
+  -webkit-box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
```
Comment (Sinetheta) [inline app/assets/stylesheets/theme/_global.scss]: Welcome to the 90's http://shouldiprefix.com/#box-shadow

File: app/assets/stylesheets/theme/_global.scss [hunk 2]
```diff
@@ -5,4 +5,12 @@
...
+  -moz-box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
+  box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
+}
+
+body {
+  background-image: image-url('bg-texture.png');
```
Comment (Sinetheta) [inline app/assets/stylesheets/theme/_global.scss]: If you change the BG image you're going to want to choose a BG color which comes "as close as possible" to the image, so the load doesn't look so shitty.

**Comment by adamish717** [comment]
Fixed drop-shadow prefix.

BG image is transparent, so can't change bg color to match.

File: app/assets/stylesheets/theme/_bootstrap-overrides.scss [hunk 3]
```diff
@@ -27,7 +27,7 @@ $coldstone:             hsl(190, 17%, 66%);
 //
 //## Settings for some of the most global styles.
 //** Background color for `<body>`.
-$body-bg:               lighten($coldstone, 25%);
+$body-bg:               #454545;
```
Comment (Sinetheta) [inline app/assets/stylesheets/theme/_bootstrap-overrides.scss]: doesn't say `$body-bg`, go ahead and put the whole background string here.

File: app/assets/stylesheets/theme/_global.scss [hunk 4]
```diff
@@ -5,4 +5,9 @@
   border-radius: $border-radius-base;
   border: 1px solid $gray-lighter;
   margin-bottom: 2em;
+    box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
```
Comment (Sinetheta) [inline app/assets/stylesheets/theme/_global.scss]: Looks like you're having some whitespace issues here.

File: app/assets/stylesheets/theme/_navbar.scss [hunk 5]
```diff
@@ -1,3 +1,8 @@
 .navbar-brand {
   padding: 10px 15px;
 }
+
+.navbar {
+  background-color: $white;
```
Comment (Sinetheta) [inline app/assets/stylesheets/theme/_navbar.scss]: Usually when you're styling Bootstrap components there's a variable to override https://github.com/adamish717/steam-hours/blob/global-add/app/assets/stylesheets/theme/_bootstrap-overrides.scss#L363

**Comment by Sinetheta** [comment]
I ended up just cherry-picking this into master and fixing it.

**Pull request closed without merging**